src/Bounty.hs

```diff
@@ -31,40 +31,22 @@ module Bounty where
 
 import           Control.Monad        hiding (fmap)
 import           Data.Aeson           (FromJSON, ToJSON)
-import           Data.List            (intersect, union)
-import qualified Data.Map             as Map
-import           Data.String          (IsString (..))
-import           Data.Text            (Text)
-import           Data.Void            (Void)
 import           GHC.Generics         (Generic)
 import           Ledger               hiding (singleton)
-import           Ledger.Ada           as Ada
-import           Ledger.Constraints   as Constraints
-import qualified Ledger.Contexts      as Validation
 import           Ledger.Credential
-import           Ledger.Index         as Index
 import qualified Ledger.Typed.Scripts as Scripts
 import           Ledger.Value         as Value
-import           Playground.Contract  (NonEmpty (..), ToSchema,
-                                       ensureKnownCurrencies, printJson,
-                                       printSchemas, stage)
-import           Playground.TH        (ensureKnownCurrencies, mkKnownCurrencies,
-                                       mkSchemaDefinitions)
-import           Playground.Types     (KnownCurrency (..))
-import           Plutus.Contract      as Contract
 import qualified PlutusTx
-import           PlutusTx.IsData
 import           PlutusTx.Maybe
 import           PlutusTx.Prelude     hiding (Semigroup (..), unless)
-import           Prelude              (Semigroup (..), Show, String, show)
-import           Text.Printf          (printf)
+import           Prelude              (Show)
 
 data Bounty = Bounty
-  { expiration           :: !POSIXTime,
-    voters               :: ![PubKeyHash],
-    requiredVotes        :: !Integer,
-    collectionMakerClass :: !AssetClass,
-    collectionToken      :: !AssetClass
+  { bExpiration           :: !POSIXTime,
+    bVoters               :: ![PubKeyHash],
+    bRequiredVotes        :: !Integer,
+    bCollectionMakerClass :: !AssetClass,
+    bCollectionToken      :: !AssetClass
```

Comment:
It's common practice to prefix record field names with data type initials. In this case one letter prefix is enough, however usually with several record data structures, two-letter prefixes are more common.

This has been one of the biggest issues with Record in Haskell. I think this particular variable scoping issue has been fixed in GHC 9.x, I've not tried it myself though so can't comment much on that.